egs/uw3/v1/run.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+stage=0
+nj=30
+color=1
+scale=40
+pad=true
+data_download=data
+data_dir=data_pad
+exp_dir=exp_pad
+
+. ./path.sh
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+. utils/parse_options.sh  # e.g. this parses the --stage option if supplied.
+
+if [ $stage -le 0 ]; then
+  # Data preparation
+  local/prepare_data.sh --dir $data_download
+fi
+
+mkdir -p $data_dir/{train,test}/data
+if [ $stage -le 1 ]; then
+  for f in train test; do
+    local/make_feature_vect.py --scale-size $scale --color $color $data_download/$f --pad $pad | \
+      copy-feats --compress=true --compression-method=7 \
+      ark:- ark,scp:$data_dir/$f/data/images.ark,$data_dir/$f/feats.scp || exit 1
+
+    steps/compute_cmvn_stats.sh $data_dir/$f || exit 1;
+  done
+fi
+
+numSilStates=4
+numStates=8
+num_gauss=10000
+
+numLeavesTri=500
+numGaussTri=20000
+numLeavesMLLT=500
+numGaussMLLT=20000
+
+boost_sil=1
+variance_floor_val=0.001
+beam=50
+
+lang_affix=${numStates}states_${numSilStates}sil
+mono_affix=${lang_affix}_${num_gauss}_var${variance_floor_val}_beam${beam}_boost${boost_sil}
+tri_affix=${mono_affix}_${numLeavesTri}_${numGaussTri}
+tri2_affix=${tri_affix}_${numLeavesMLLT}_${numGaussMLLT}
```

Comment:
I know all these affixes have been helpful while doing the experiments,
but it's best if you remove them all so that your final directories are like
`exp/mono` and `exp/tri2`, `data/lang`, etc. Make the necessary changes in the chain recipe too.
Also, please remove variance_floor_val.
@aarora8 please do the same for your PR.